src/auro-popover.js

```diff
@@ -89,28 +78,28 @@ class AuroPopover extends LitElement {
     handleHide = () => {
       this.toggleHide();
     },
-    handleTabWhenFocusOnTrigger = (event) => {
-      if (event.key.toLowerCase() === 'tab') {
-        this.toggleHide();
-      }
-    };
+    handleKeyboardWhenFocusOnTrigger = (event) => {
+      const key = event.key.toLowerCase();
 
-    if (!this.sticky) {
-      this.trigger.addEventListener('touchstart', handleShow);
-    }
+      if (this.isPopoverVisible) {
+        if (key === 'tab' || key === 'escape') {
+          this.toggleHide();
+        }
+      }
 
-    if (this.sticky) {
-      this.trigger.addEventListener('click', handleShow);
-    } else {
+      if (key === ' ' || key === 'enter') {
+        this.toggle();
```

Comment:
this should fire when the trigger has focus

Reply:
I don't think changing focus is the only case in which the content should be dismissed. These key presses allow the user finer control.

Reply:
This is the model we are going after. https://shoelace.style/components/tooltip and https://material-ui.com/components/tooltips/

When the user tabs to the element, the popover should appear, alert voice over, read the content and when the user tabs away it will close. 

Allowing the user additional keyboard control is not common across different libraries.

Reply:
I like the idea of auto-show on focus. Adding that in.

The documentation you linked does actual show support for manual hide/show of the tooltip, albeit in their library nesting that behind a property flag.